backend/bitboxbase/updater/config.go

```diff
@@ -0,0 +1,92 @@
+// Copyright 2018 Shift Devices AG
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package updater contains the API to the physical device.
+package updater
+
+import (
+	"bytes"
+
+	"github.com/digitalbitbox/bitbox-wallet-app/util/config"
+	"github.com/flynn/noise"
+)
+
+const configFilename = "base.json"
+
+type noiseKeypair struct {
```

Comment:
the whole file should eventually be refactored/moved to util/, so it can be shared with the bitbox02 implementation. e.g. `util/noiseconfig`